src/main/java/cashleh/CashLeh.java

```diff
@@ -30,8 +30,7 @@ public void run() {
                     + " / /___/ /_/ (__  ) / / / /___/  __/ / / /_/\n"
                     + " \\____/\\__,_/____/_/ /_/_____/\\___/_/ /_(_)\n";
         String userGuideLink = ("Here is the link to the user guide:"
-                + "https://docs.google.com/document/d/"
-                + "15h45BB5kMkTZ6bkwUHujpYwxVVl80tNEyNUsEVyk5AQ/edit?usp=drive_link");
+                + "https://ay2324s1-cs2113-w12-2.github.io/tp/UserGuide.html");
```

Comment:
Maybe we could insert a line break before displaying the link, in order to increase its visibility and make it easier to copy.